apps/e2e/cypress/e2e/admin/pool-candidate.cy.js

```diff
@@ -86,6 +87,19 @@ describe("Pool Candidates", () => {
       cy.getMe().then((testUser) => {
         cy.get("@publishedTestPool").then((pool) => {
           cy.createApplication(testUser.id, pool.id).then((poolCandidate) => {
+            cy.getMeAllData().then((me) => {
```

Comment:
This block of code has been copy-pasted throughout the tests so it's probably a good candidate for a Cypress command.  Or maybe just roll it into `createApplication`.

Reply:
I was iffy on it as I didn't want to hide or abstract too much of what was going on
I put it into `createApplication` with some comments explaining things
https://github.com/GCTC-NTGC/gc-digital-talent/pull/6756/commits/cd25fec21d31a1b36d505041d6954b6e09fdda52

Reply:
Yeah, nice. :+1: